mms/client_sdk_generator.py

```diff
@@ -25,34 +25,33 @@ class ClientSDKGenerator(object):
     Class for client SDK Generator
     """
     @staticmethod
-    def generate(openapi_endpoints, sdk_lanugage):
+    def generate(openapi_endpoints, sdk_language):
```

Comment:
We should revisit this:
1. We have upgraded to openapi 3.0, we should get rid of swager 2.0
2. the json will be generated by java code. we should use openapi library to create json file.
3. It's better have a single java entry to create a json and call swager 3.0 to generate the client package
4. We should revisit the use cases:
a. only generate client for single model entry point
b. generate only model management APIs
c. should we generate deprecated APIs?
d. Can we generate multiple models APIs?